app/views/ui-components/v1/cards/_metal_level_select.html.slim

```diff
@@ -259,36 +262,49 @@ javascript:
         if (plan.network_information)
           network = `NETWORK NOTES <a data-toggle="tooltip" data-placement="top" data-container="body" title="${plan.network_information}"><i class="fas fa-question-circle"></i></a>`
           var innerHTML =
-          `<div class="col-xs-4 reference-plans">
+          `<div class="col-xs-4 reference-plans plan-tile">
             <div class="col-xs-12 p0 mb-1">
               <label class="container">
-                <p class="heading-text reference-plan-title mb-1"> ${plan.title}</p>
-                <span class="plan-label">Type:</span> <span class="rp-plan-info" id="plan-type-value">${plan.product_type}</span><br>
-                <span class="plan-label"">Carrier:</span> <span class="rp-plan-info" id="plan-carrier-value">${plan.carrier_name}</span><br>
-                <span class="plan-label">Level:</span> <span class="rp-plan-info" id="plan-level-value">${plan.metal_level_kind}</span><br>`;
+                <p class="heading-text reference-plan-title mt-1"> ${plan.title}</p>
+                <div class="row mt-n2">
+                  <div class="${(plan.is_pvp_eligible && isPvpFeatureEnabled === 'true') ? 'pvp-ref-plan-indicator' : 'hidden'}">
+                    <i class="fas fa-lg fa-square" data-toggle="tooltip" data-placement="top" data-container="body" title="${pvpTooltipValue}"></i> Premium Value Plan
+                  </div>
+                  <div class="${plan.is_standard_plan ? (isPvpFeatureEnabled === 'true' ? 'standard-ref-plan-indicator' : 'text-uppercase') : 'hidden'} ml-1" style="${isPvpFeatureEnabled === 'true' ? '' : 'color:green'}">
+                    <i class="${isPvpFeatureEnabled === 'true' ? 'fas fa-lg fa-square' : 'fas fa-bookmark'}" data-toggle="tooltip" data-placement="top" data-container="body" title="${standardPlanTooltipValue}"></i> Standard Plan
```

Comment:
This appears unique in that we _do_ use a square for standard plans here, instead of the bookmark icon as elsewhere (including https://github.com/health-connector/enroll/pull/2759/files#diff-f61cbe01c174b03b07eb8d251393d6c070ca7fdefe953ff0860e8fc0daf08c96R14). Whether correct or not, I don't know, but just flagging for verification.